diff --git a/configure b/configure
index 086c70ec..bee37f59 100755
--- a/configure
+++ b/configure
@@ -3810,8 +3810,8 @@ case $host_os in *\ *) host_os=`echo "$host_os" | sed 's/ /-/g'`;; esac
 
 
 if test "$cross_compiling" != "no"; then
-    HOST_PREFIX="${host_alias}-"
-    HOST_SUFFIX="-$host_alias"
+    HOST_PREFIX=
+    HOST_SUFFIX=
 else
     HOST_PREFIX=
     HOST_SUFFIX=
@@ -45710,4 +45710,3 @@ else
     echo "no"
 fi
 echo ""
-
diff --git a/wx-config.in b/wx-config.in
index dd864605..9f3a16b8 100755
--- a/wx-config.in
+++ b/wx-config.in
@@ -398,7 +398,7 @@ get_mask()
 }
 
 # Returns true if this script is for a cross compiled config.
-is_cross()  { [ "x@cross_compiling@" = "xyes" ]; }
+is_cross()  { [ "xno" = "xyes" ]; }
 
 
 # Determine the base directories we require.
@@ -408,7 +408,7 @@ wxconfdir="@libdir@/wx/config"
 
 installed_configs=`cd "$wxconfdir" 2> /dev/null && ls | grep -v "^inplace-"`
 
-is_cross && target="@host_alias@"
+is_cross && target=
 
 # Define a pseudo-hash to contain the specification of this wx-config
 # instance and its associated library.
