diff --git a/configure b/configure
index 670ed2b9a4..05a0e6fced 100755
--- a/configure
+++ b/configure
@@ -3849,7 +3849,7 @@ case $host_os in *\ *) host_os=`echo "$host_os" | sed 's/ /-/g'`;; esac
 
 if test "$cross_compiling" != "no"; then
     HOST_PREFIX="${host_alias}-"
-    HOST_SUFFIX="-$host_alias"
+    HOST_SUFFIX=
 else
     HOST_PREFIX=
     HOST_SUFFIX=
diff --git a/configure.in b/configure.in
index fbded05cce..21d805e209 100644
--- a/configure.in
+++ b/configure.in
@@ -28,7 +28,7 @@ dnl only set to "maybe" and not "yes" and will be either set to "yes" later or
 dnl configure will exit with error in AC_PROG_CC so don't test for = "yes" here
 if test "$cross_compiling" != "no"; then
     HOST_PREFIX="${host_alias}-"
-    HOST_SUFFIX="-$host_alias"
+    HOST_SUFFIX=
 else
     HOST_PREFIX=
     HOST_SUFFIX=
